Tray menu is not working on Linux

Tray menu should contain items such as "Punch Time", "Show App" and "Quit". But in Linux, it only shows "Time to Leave", and clicking on it just displays the app.

Reproduced in Ubuntu, but I expect other Linux distributions to have the same issue.

To reproduce:
- Open the App
- Minimize it
- See the icon on the tray
- Right-click on the icon

@thamara could explain how to build for linux? I'm on ubuntu and ran `package:deb`, went to the release folder and tried running `time-to-leave` by command line but get errors like: `Cannot find module 'i18next'`.
Then I notice that package is in devDependencies, so I tried putting in dependencies in package.json, then I get this error: `TypeError: Cannot read property 'setMenu' of null`

Hi @lucas-barros, `npm run package:deb` will create the package we distribute on the releases. For development, there's no need to do this.
You can follow the steps in our [development guide](https://github.com/thamara/time-to-leave/blob/main/DEVELOPMENT.md), but basically, it's just calling `npm start`, and it will run the app. Be aware it's possible it will startup minimized on the tray.

Regarding the `Cannot find module 'i18next'.`, indeed this is an error that I forgot to push the fix. Will fo this now.
The `TypeError: Cannot read property 'setMenu' of null` is something new, which might even be related to this issue.. I'll try to reproduce this error and will open another issue for it. 
Can you please confirm that you can still reproduce this issue using `npm start`?

--- a/js/main-window.js
+++ b/js/main-window.js
@@ -93,6 +93,7 @@ function createWindow()
         let contextMenuTemplate = getContextMenuTemplate(mainWindow);
         contextMenuTemplate[0].enabled = arg;
         contextMenu = Menu.buildFromTemplate(contextMenuTemplate);
+        tray.setContextMenu(contextMenu);
     });
 
     ipcMain.on('RESIZE_MAIN_WINDOW', (event, width, height) =>

